configs/annexes/amplifybackend.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Should this be called amplifyadminui? https://docs.aws.amazon.com/amplify-admin-ui/latest/APIReference/what-is-admin-ui.html

Reply:
Nah, it's called amplifybackend in the aws cli.

Reply:
Some of the older services are named deliberately to try and make the AWS product vs documentation vs actual service name more sensible, but It doesn't scale - I think just rolling with the botocore name (ie. what ends up in the CLI) is "the one true way", even if a little nonsensical sometimes.

Reply:
Yep, I agree with that.